src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/CodeActionEndpointTest.cs

```diff
@@ -24,6 +24,7 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.CodeActions
 {
     public class CodeActionEndpointTest : LanguageServerTestBase
     {
+        private readonly RazorDocumentMappingService DocumentMappingService = Mock.Of<RazorDocumentMappingService>();
```

Comment:
Tests updated to a passing state. Testing of functionality introduced with this PR is to be done.

I'd like to get this merged first to ensure the PR isn't too large. I'll subsequently implement the resolve endpoint, and do the testing for it all together because there's likely to be a good deal of overlap.